backend_simple.py

```
@@ -51,7 +65,18 @@
 )
 
 # Security
-security = HTTPBearer()
+security = HTTPBearer(auto_error=False)  # Make bearer token optional
+
+# OAuth state tracking for CSRF protection with TTL
+from time import time
```

Ruff (0.12.2)

Redefinition of unused `time` from line 10

(F811)